src/main/flight/mixer.c

```diff
@@ -355,7 +355,7 @@ void initEscEndpoints(void)
 void mixerInitProfile(void)
 {
 #ifdef USE_DYN_IDLE
-    if (motorConfig()->dev.useDshotTelemetry) {
+    if (isRpmFilterEnabled()) {
```

Comment:
Should we move this into the config validation to make it more obvious to users that this is happening?

Reply:
I thought about that, but all we can do is reset the `idle_min_rpm` to 0 effectively losing their setting. Then if they fix it by enabling RPM filter the value will still be at 0 and not work so kind of a catch-22. It's not good either way.

Reply:
At least that gives the user some feedback indicating that the configuration is not valid - in most cases they will spot it when checking the settings after a 'Save & Reboot', as this is the pattern that is used for a lot of other settings.
Disabling things at runtime based on dependencies on other settings makes it very hard to diagnose.

Reply:
Ok, makes sense. Lesser of two evils I guess. I'll change it to disabling in the config check.

Reply:
Yeah. Resetting stuff that is not supported is not good UX. But in my opinion it is better UX than silently not doing what the user configured it to do - in non-obvious cases like the one at hand this will be sending users on a wild goose chase of test flights trying to figure out the right setting this before they go and report this as not working.